reports/utils.py

```diff
@@ -0,0 +1,34 @@
+from math import asin, cos, radians, sin, sqrt
+
+LINKED_EVENTS_GEOS_PROJECTION_SRID = 3067
+
+"""
+FIXME: get_distance would be better alternative to haversine(),
+since it calculates the 3d distance.
+Can the 3d distance really be twice the 2d distance or why is the number so much bigger?
+"""
+# from django.contrib.gis.measure import Distance
```

Comment:
As pointed out earlier, this would need gdal libraries. So you might neet to do sometthing like this:
- https://github.com/City-of-Helsinki/maritime-maas/blob/main/.github/workflows/ci.yml#L58
- https://github.com/City-of-Helsinki/maritime-maas/blob/main/Dockerfile#L13-L14

Reply:
Too much for too little at this time, I think. We could consider this later, but now I self prefer that we go with 2d model. The difference won't even matter in our use.

Reply:
But still, nice to know how to get it to work.